cli/pkg/k8s/k8s.go

```diff
@@ -543,4 +543,14 @@ func GetAllPVCSizes() (map[string]string, error){
 		results[pvc.Name] = size.String()
 	}
 	return results, nil
+}
+
+func GetUpdatedPromscaleMemResource() (string, error) {
```

Comment:
nit: should this be in cli/tests instead?

Reply:
We are maintaining k8s utilities used in e2e tests here in the bottom with a heading. As this is growing now. I will separate them to `cli/tests` directory.